style/scss/_hero.scss

```diff
@@ -57,4 +57,50 @@
       }
     }
   }
-}
\ No newline at end of file
+}
+
+@media screen and (max-width: $phone-size) {
+  .hero {
+    max-height: 471px;
+    padding-top: 63px;
+    padding-left: 53px;
+    padding-right: 53px;
+    background-size: 280% 150%;
+    background-position: bottom -35px right -115px;
+    
+    &__content {
+      width: 245px;
+      height: 236px;
+      gap: 20px;
+  
+      &__title {
+        width: 245px;
+        color: $black;
+        font-size: 20px;
+        font-weight: 500;
+        line-height: 112%;
+        text-align: center;
+      }
+  
+      &__description {
+        width: 243px;
```

Comment:
Please try to remove the width and height for all the elements, and use padding instead.